flink-formats/flink-parquet/src/main/java/org/apache/flink/formats/parquet/utils/NestedPositionUtil.java

```diff
@@ -50,12 +50,10 @@ public static RowPosition calculateRowOffsets(
         int nullValuesCount = 0;
         BooleanArrayList nullRowFlags = new BooleanArrayList(0);
         for (int i = 0; i < fieldDefinitionLevels.length; i++) {
+            // If a row's last field is an array, the repetition levels for the array's items will be
+            // larger than the parent row's repetition level, so we need to skip those values.
```

Comment:
A lint issue. Sorry.
```suggestion
            // If a row's last field is an array, the repetition levels for the array's items will
            // be larger than the parent row's repetition level, so we need to skip those values.
```